### PR TITLE
Fixes/v0.11.0 post

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Proposed changes

This PR fixes multiple issues observed during v0.11.0 release

## Fixes

-

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

-

### CODE REVIEW

File: plugins/aea-ledger-cosmos/setup.py [hunk 1]
```diff
@@ -50,7 +50,7 @@
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
-        "Framework :: AEA" "Topic :: Communications",
```
Comment (DavidMinarsch) [inline plugins/aea-ledger-cosmos/setup.py:53]: this specifier did not work, even with fixed typo

Comment (marcofavorito) [reply plugins/aea-ledger-cosmos/setup.py:53]: ok

**Review by marcofavorito — APPROVED** [review]

File: aea/cli/create.py [hunk 2]
```diff
@@ -181,6 +182,7 @@ def _create_agent_config(ctx: Context, agent_name: str, set_author: str) -> Agen
         description="",
         default_ledger=DEFAULT_LEDGER,
         default_connection=None,
+        dependencies=dependencies_from_json(DEFAULT_DEPENDENCIES),
```
Comment (DavidMinarsch) [inline aea/cli/create.py:185]: Without this, there's a bit too many config set steps

File: docs/tac-skills-contract.md [hunk 3]
```diff
@@ -154,17 +154,17 @@ In a separate terminal, in the root directory, fetch at least two participants:
 ``` bash
 aea fetch fetchai/tac_participant:0.23.0 --alias tac_participant_one
 cd tac_participant_one
-aea generate-key ethereum
-aea add-key ethereum ethereum_private_key.txt
 aea install
 aea build
```
Comment (DavidMinarsch) [inline docs/tac-skills-contract.md:158]: Note, the order now matters!

File: packages/fetchai/agents/aries_alice/aea-config.yaml [hunk 4]
```diff
@@ -31,4 +31,6 @@ default_routing:
   fetchai/http:0.13.0: fetchai/http_client:0.18.0
   fetchai/oef_search:0.14.0: fetchai/soef:0.18.0
 connection_private_key_paths: {}
-dependencies: {}
+dependencies:
```
Comment (DavidMinarsch) [inline packages/fetchai/agents/aries_alice/aea-config.yaml:34]: All agents need at least one ledger deps, otherwise, they cannot be run.

**Review by marcofavorito — APPROVED** [review]

**Review by 5A11 — APPROVED** [review]
lgtm

**Pull request merged**